activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -361,15 +387,7 @@ def columns(table_name) #:nodoc:
       # Returns an array of indexes for the given table.
       def indexes(table_name, name = nil) #:nodoc:
         exec_query("PRAGMA index_list(#{quote_table_name(table_name)})", 'SCHEMA').map do |row|
-          sql = <<-SQL
-            SELECT sql
-            FROM sqlite_master
-            WHERE name=#{quote(row['name'])} AND type='index'
-            UNION ALL
-            SELECT sql
-            FROM sqlite_temp_master
-            WHERE name=#{quote(row['name'])} AND type='index'
-          SQL
+          sql = "SELECT sql FROM sqlite_master WHERE name=#{quote(row['name'])} AND type='index'"
```

Comment:
I removed it because `alter_table` is no longer using temporary tables. However, I should and will put it back in; I had forgotten that temporary tables can also be created directly by the user.